maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -103,94 +93,11 @@ private void testing(File runFile) throws MojoExecutionException, Exception, IOE
 			}
 
 			if (resolve) {
-				resolve(run);
+				run.resolve(failOnChanges, false);
```

Comment:
We have the same issue here since resolve is called with false. How is the resolve output used by the following test execution?

Also need to check for and report errors/warnings.